modules/oe_translation_epoetry/src/Plugin/RemoteTranslationProvider/Epoetry.php

```diff
@@ -148,6 +159,23 @@ public function submitConfigurationForm(array &$form, FormStateInterface $form_s
   public function newTranslationRequestForm(array &$form, FormStateInterface $form_state): array {
     $languages = $this->languageManager->getLanguages();
     $entity = $this->getEntity();
+
+    // Check if we have already made requests to ePoetry for this node, because
+    // if we did, we need to use a different request type and inform the user.
+    // To determine the last request, we first check the form state as it may
+    // already be set by NewVersionCreateForm in case this is a request during
+    // an ongoing one.
+    $last_request = $form_state->get('epoetry_last_request');
+    if (!$last_request) {
+      $last_request = $this->getLastRequest($entity);
+    }
+    if ($last_request) {
+      $form_state->set('epoetry_last_request', $last_request);
+      $form['info'] = [
+        '#markup' => $this->t('You are making a request for a new version. The previous version was translated with the <strong>@id</strong> request ID.', ['@id' => $last_request->getRequestId(TRUE)]),
```

Comment:
Wouldn't it make sense to have the languages from the previous version already checked since we can access the request?

Reply:
Boh, not for now, the two are not necessarily correlated. We can see if COMM says they want that.